rpc/rpc-transport/socket/src/name.c

```diff
@@ -108,27 +108,20 @@ af_unix_client_bind(rpc_transport_t *this, struct sockaddr *sockaddr,
     path_data = dict_get_sizen(this->options, "transport.socket.bind-path");
     if (path_data) {
         char *path = data_to_str(path_data);
```

Comment:
Do it before  line number 109. data_to_str already checks if path_data will be NULL or not and it returns value accordingly.
On line number  109 use if (!path_data && !path && path_data->len > 108)

Reply:
I think so this change is already outdated. The new change is using strncpy.